.env.example

```diff
@@ -6,3 +6,5 @@ DB_PASSWORD=postgres
 DB_IMAGE_TAG=latest
 # Default will likely be "latest" but can also be any tag available for the image hosted at https://ghcr.io/orcacollective/spd-lookup-api
 API_IMAGE_TAG=latest
+# Roster source can be used to point at a local source for rosters for testing purposes, production should always point to the default value below
+ROSTER_SOURCE=https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup
```

Comment:
We should make this default NOT our actual production 😅  Probably just a shim value.

Reply:
The prod source here is just a read only source, so I figured pointing people there by default was the best bet for "pull this down and spin it up if you want to" scenarios.

Reply:
That's fair...and I guess we have techbloc references elsewhere 🤷🏼‍♀️

Reply:
Oh but wait, should this have `/latest` on the end like the other reference?

Reply:
ye it's updated I think we're just commenting on an out of dated bit of code.

Reply:
🤦🏼‍♀️ that's what I get for multitasking, sorry!